frontend/src/components/SlideSearchFunction.css

```diff
@@ -0,0 +1,41 @@
+.noEntrySearch,
+.enterYourSearch {
+  display: none;
+}
+
+@media screen and (max-width: 600px) {
+  .searchBarButton {
+    margin-left: 0;
+    z-index: 2;
```

Comment:
I don't really like the `z-index` property, it's really easy to do some really bad code with it. Could you explain to me why you did use it ? (It's not a rebuttal, but I want to be sure you understand what you did)

Reply:
i used the Z-index, because : I wanted that the button stay over the popup, when the popup is open. but after reflection I decide to go down the PopUp. so it's true I don't need z-index now. i'll delete it .

Reply:
z-index is deleted ;)